ddpui/tests/services/test_dbt_service.py

```
@@ -1,18 +1,22 @@
 from ddpui.models.org import Org, OrgDbt, OrgPrefectBlockv1, OrgWarehouse
+from ddpui.models.canvas_models import CanvasNode, CanvasNodeType, CanvasEdge
+from ddpui.core.orgdbt_manager import DbtCommandError
 from ddpui.ddpdbt.dbt_service import (
     delete_dbt_workspace,
     setup_local_dbt_workspace,
+    parse_dbt_manifest_to_canvas,
+    generate_manifest_json_for_dbt_project,
 )
 from ddpui.ddpprefect import DBTCLIPROFILE, SECRET
-from ddpui.ddpdbt.schema import DbtProjectParams
+from ddpui.core.orgdbt_manager import DbtCommandError
```

Ruff (0.14.10)

Redefinition of unused `DbtCommandError` from line 11

Remove definition: `DbtCommandError`

(F811)